### PR TITLE
Wait and version fixes

### LINKED ISSUE #5407: Use explicit wait to check 'Plugin available' option on the OCP console
Test failure is seen on RUN ID: 1643402311

### PR DESCRIPTION
**Pull request by am-agrawa**
fixed #5407 

Signed-off-by: Aman Agrawal <amagrawa@redhat.com>

### CODE REVIEW

File: ocs_ci/ocs/ui/validation_ui.py [hunk 1]
```diff
@@ -179,7 +179,8 @@ def odf_console_plugin_check(self):
         """
         self.ocp_version = get_ocp_version()
         ocs_version = version.get_semantic_ocs_version_from_config()
-        if self.ocp_version >= "4.9" and ocs_version >= version.VERSION_4_9:
+        ocp_version = version.get_semantic_ocp_version_from_config()
+        if ocp_version >= version.VERSION_4_9 and ocs_version >= version.VERSION_4_9:
```
Comment (OdedViner) [inline ocs_ci/ocs/ui/validation_ui.py:183]: You can use `self.ocp_version_semantic`
https://github.com/red-hat-storage/ocs-ci/blob/447fc6c4eb54ba038fc613eb8a1df32577140d5a/ocs_ci/ocs/ui/base_ui.py#L342

Comment (am-agrawa) [reply ocs_ci/ocs/ui/validation_ui.py:183]: Is the current implementation incorrect?

Comment (am-agrawa) [reply ocs_ci/ocs/ui/validation_ui.py:183]: I think it's exactly the same except for variable name

Comment (OdedViner) [reply ocs_ci/ocs/ui/validation_ui.py:183]: We have var in `Page_Nav` class. so we can use it..[OOP inheritance]
https://www.geeksforgeeks.org/inheritance-in-python/

**Review by OdedViner — CHANGES_REQUESTED** [review]
Pleas use `self.ocp_version_semantic` var

**Review by OdedViner — APPROVED** [review]

**Review by jilju — APPROVED** [review]

**Review by vavuthu — APPROVED** [review]

**Pull request merged**